bootstrap/handlers/httpserver/httpserver.go

```diff
@@ -77,7 +77,12 @@ func (b *HttpServer) BootstrapHandler(
 	}
 
 	bootstrapConfig := container.ConfigurationFrom(dic.Get).GetBootstrap()
-	addr := bootstrapConfig.Service.Host + ":" + strconv.Itoa(bootstrapConfig.Service.Port)
+
+	// Do not use Service.Host in the address.
+	// Using hostname it is not recommended because it will create a listener for at most one of the host's IP addresses
+	// which becomes an issue when using distributed orchestration.
+	addr := ":" + strconv.Itoa(bootstrapConfig.Service.Port)
```

Comment:
Wouldn't a default "0.0.0.0" IP address be more explicit here?

Reply:
And along those lines, couldn't Service.Host be set to "0.0.0.0" in the configuration file and any code change be avoided?  Changing this code in this way precludes the anything but a wide-open listen.  Are you sure this is desired in all cases?

Reply:
> Wouldn't a default "0.0.0.0" IP address be more explicit here?

Yes, probably a good idea

Reply:
> And along those lines, couldn't Service.Host be set to "0.0.0.0" in the configuration file and any code change be avoided? Changing this code in this way precludes the anything but a wide-open listen. Are you sure this is desired in all cases?

No, because it is also used for service registry, which breaks Consul health checks.

This is the quick fix. The longer term fix is to add a separate config setting just for this which allows wide-open (0.0.0.0) or locking it down to a specific adapter.

Reply:
Changed to use `0.0.0.0` IP address